File: exercises-cpp/3. C++/1. Secuenciales/15.cpp
#include <iostream>
#include <cstdio>
#include <cmath>
using namespace std;

int main()
{
    system("cls");
    
    float juan,rosa,daniel,total,juanPorcen,rosaPorcen,danielPorcen;
    printf("Juan: "); cin>>juan;
    printf("Rosa: "); cin>>rosa;
    printf("Daniel: "); cin>>daniel;

    daniel = daniel / 3;
    total = juan + rosa + daniel;

    juanPorcen = 100 * juan / total;
    rosaPorcen = 100 * rosa / total;
    danielPorcen = 100 * daniel / total;

    cout<<"Total: "<<total<<" dolares."<<endl;
    cout<<"Aporte de Juan (en porcentaje): "<<juanPorcen<<endl;
    cout<<"Aporte de Rosa (en porcentaje): "<<rosaPorcen<<endl;
    cout<<"Aporte de Daniel (en porcentaje): "<<danielPorcen<<endl;
}

File: exercises-python/1. Problemas secuenciales/15.py
'''
Juan, Rosa y Daniel aportan cantidades de dinero para formar un capital.
Juan y Rosa aportan en dólares y Daniel, en soles. Desarrolle el programa
que determine el capital total en dólares y que porcentaje de dicho capital
aporta cada uno. Dólar = S/. 3.00 nuevos soles.
'''

import os
os.system("cls")

#Datos
juan = int(input("Aporte de Juan: ")) #Dólares
rosa = int(input("Aporte de Rosa: ")) #Dólares
daniel = int(input("Aporte de Daniel: ")) #Soles

#Conversión
daniel = daniel / 3

#Total
total = juan + rosa + daniel

#Porcentaje
juanPorcen = 100 * juan / total
rosaPorcen = 100 * rosa / total
danielPorcen = 100 * daniel / total

#Salida
print(f"Total: {total:.2f} dólares.")
print(f"Aporte de Juan (en porcentaje): {juanPorcen:.2f}")
print(f"Aporte de Rosa (en porcentaje): {rosaPorcen:.2f}")
print(f"Aporte de Daniel (en porcentaje): {danielPorcen:.2f}")